### PR TITLE
Change/Upgrade default values for image related paths.

### PR DESCRIPTION
**Pull request by maethu**
- Change location of marker.png according collective.geo.openlayers 4.0
 - Change location of theme resources according collective.geo.openlayers 4.0

Support Plone 4/5

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,11 @@ Changelog
 3.2 (unreleased)
 ----------------
 
+- Change/Upgrade default values for image related paths [mathias.leimgruber]
+  - Change location of marker.png according collective.geo.openlayers 4.0
```
Comment (jone) [inline docs/HISTORY.txt]: restructured text requires an empty line obove this line in order to detect the nested list correctly 😉

File: src/collective/geo/settings/upgrades/configure.zcml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    i18n_domain="collective.geo.settings">
+
+  <genericsetup:upgradeStep
+      title="Set new default value for plone 5"
+      source="31"
+      destination="32"
+      handler="collective.geo.settings.upgrades.migrate_default_marker_image.run"
```
Comment (jone) [inline src/collective/geo/settings/upgrades/configure.zcml]: I would probably rename the module to something like `v23_ migrate_default_marker_image` in order to have a better naming system for when more upgrade steps are added.

**Review by jone — APPROVED** [review]
Looks good 👍

File: src/collective/geo/settings/upgrades/configure.zcml (hunk 2 above)
Comment (maethu) [reply src/collective/geo/settings/upgrades/configure.zcml]: I thought about that too... thanks I change that.

**Pull request merged**